curriculum/templates/curriculum/lessonplan_find_share.html

```diff
@@ -67,53 +72,63 @@ <h1 class="font-weight-bold find-share-curricula">Find & Share Curricula</h1>
-                            <p class="stars" data-average-rating="{{ lessonplan.average_rating }}" >
-                                <i class="fas fa-star" aria-hidden="true"></i>
-                                <i class="fas fa-star" aria-hidden="true"></i>
-                                <i class="fas fa-star" aria-hidden="true"></i>
-                                <i class="fas fa-star" aria-hidden="true"></i>
-                                <i class="fas fa-star" aria-hidden="true"></i>
-                            </p>
-                        {% endif %}
-                        {% if lessonplan.web_only %}
-                            <p class="pr-4" title="Chromebook compatible"><i class="text-right fab fa-chrome"></i></p>
-                        {% endif %}
+                <a style="color: rgb(0,0,0); text-decoration:none" href="{{ lessonplan.get_absolute_url }}">
+                <span>
+                    <div class="col-4">
+                        <div class="d-flex flex-column curriculum mb-3 border rounded border-primary p-3">
+                            <h6 class="curriculum-title text-primary">{{ lessonplan.title }}</h6>
+                            <div class="d-flex flex-row justify-content-between">
+                                {% if lessonplan.average_rating %}
+                                    <p class="stars text-primary" data-average-rating="{{ lessonplan.average_rating }}">
+                                        <i class="fa-star" aria-hidden="true"></i>
+                                        <i class="fa-star" aria-hidden="true"></i>
+                                        <i class="fa-star" aria-hidden="true"></i>
+                                        <i class="fa-star" aria-hidden="true"></i>
+                                        <i class="fa-star" aria-hidden="true"></i>
+                                    </p>
+                                {% else %}
+                                    <p class="stars" data-average-rating="{{ lessonplan.average_rating }}">
+                                        <i class="fas fa-star" aria-hidden="true"></i>
+                                        <i class="fas fa-star" aria-hidden="true"></i>
+                                        <i class="fas fa-star" aria-hidden="true"></i>
+                                        <i class="fas fa-star" aria-hidden="true"></i>
+                                        <i class="fas fa-star" aria-hidden="true"></i>
+                                    </p>
+                                {% endif %}
+                                {% if lessonplan.web_only %}
+                                    <p class="pr-4" title="Chromebook compatible"><i
+                                            class="text-right fab fa-chrome"></i></p>
+                                {% endif %}
+                            </div>
+                            <h6 class="curriculum-text">{{ lessonplan.summary }}</h6>
```

Comment:
This should still be a `p` tag, not `h6`

Reply:
Soo leaving it as a p tag turns the description font size to be much smaller than what it was initially before it was in the anchor tag. If you want me to change it back I can

Reply:
Oh I see. The `curriculum a` class is overriding the entire card. I think we should keep the text as it was before, so I'd suggest either changing the `curriculum-text` class to specify `font-size=1rem` OR changing the `curriculum a` class to be something more specific like `curriculum-link` and using that only for the `View more details` tag.

Reply:
Okay I will try and implement that!

Reply:
Done! You should see that the curriculum text is a `p` element now and I added a CSS rule for `curriculum-text`. I noticed it was already a class being referenced, but couldn't find it in _main.css_. Let me know you're thoughts!